.rhdh/bundle/manifests/rhdh-operator.csv.yaml

```diff
@@ -42,7 +42,7 @@ metadata:
     features.operators.openshift.io/token-auth-gcp: "false"
     repository: https://gitlab.cee.redhat.com/rhidp/rhdh/
     support: Red Hat
-    skipRange: '>=1.0.0 <1.3.2'
+    skipRange: '>=1.2.0 <1.3.2'
```

Comment:
while valid, why do we need to change this?

Reply:
Starting from 1.0.0 means a direct update from, say, 1.1.0 to 1.3.2, no? Could this be an issue, knowing the possible breaking changes that might happen across minor versions of the operand? Would it make more sense to allow some kind of "progressive" minor upgrades?

Reply:
Yes, direct updates from earlier versions like 1.1.0 to 1.3.2 could introduce issues. To address this, I am maintaining the replaces field 1.2.5 -> 1.3.2 and ensuring the skipRange limits upgrades to versions within the 1.2.x series.